core/domain/learner_progress_services_test.py

```diff
@@ -18,13 +18,18 @@
 
 import datetime
 
+from constants import constants
 from core.domain import collection_domain
 from core.domain import collection_services
 from core.domain import exp_services
 from core.domain import learner_playlist_services
 from core.domain import learner_progress_services
+from core.domain import rights_manager
+from core.domain import user_services
 from core.platform import models
 from core.tests import test_utils
+import feconf
+
```

Comment:
Do you need this extra line?

Reply:
I was getting a linting error for it, but I believe it was actually because import feconf should have gone after import datetime to keep the imports together. I'll fix that!